test/e2e/node/crictl.go

```diff
@@ -33,42 +33,28 @@ var _ = SIGDescribe("crictl", func() {
 	ginkgo.BeforeEach(func() {
 		// `crictl` is not available on all cloud providers.
 		e2eskipper.SkipUnlessProviderIs("gce", "gke")
-		// The test requires $HOME/.ssh/id_rsa key to be present.
```

Comment:
Not commenting on the removals but the line above.  Is there some less-tied-to-cluster/-way of knowing whether `crictl` is available?

Reply:
(crictl doesn't seem like a cloud-provider implementation detail)

Reply:
Hm, how would we do that? Could we say every node should have `crictl` configured? (tending to no)

Reply:
Honestly if crictl can't reasonably be expected to exist by default on all clusters, this should be tagged with [Feature:CrictlInstalled] or something. I'll leave it be since this is how this has always been, but we should reconsider this for v1.23 or if we ever want to untangle from kube-up.sh